spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,114 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+
+  def create_user
+    @user = User.create(first_name: "Jon", last_name: "Snow", email: "lordcommander@thewall.we")
+  end
```

Comment:
You should read and learn about ```let!(:user) ....``` style syntax for rspec.

https://www.relishapp.com/rspec/rspec-core/v/2-5/docs/helper-methods/let-and-let